### PR TITLE
Hometask 3

### PR DESCRIPTION
**Pull request by GordinRoman**

### CODE REVIEW

File: Story-1/__tests__/Subtask-1.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  });
+  describe('function strLengthSum', function() {
+    it('takes two lines and returns the sum of their lengths', function() {
+      expect(strLengthSum('Roman', 'do')).toEqual(7);
+      expect(strLengthSum('something', 'useful')).toEqual(15);
+      expect(strLengthSum('right', 'now')).toEqual(8);
+    });
+  });
+});
+
```
Comment (SergeiReutov) [inline Story-1/__tests__/Subtask-1.test.js]: Совет по написанию тестов: старайся проверять крайние значения (по-английски это называется edge cases, не знаю как правильно перевести на русский).
В случае со строкой edge case - пустая строка.
То есть можно проверить, что
```javascript
expect(strLengthSum('str', '')).toEqual(3);
expect(strLengthSum('', '')).toEqual(0);
```

В случае с цифрами стоит всегда проверять 0 и отрицательные значения.
Ну и так далее. Короче, старайся как бы обхитрить свою функцию, чтобы она сработала неправильно.

File: Story-2/Subtask-2.js [hunk 2]
```diff
@@ -38,41 +36,40 @@ console.log(obj); // вернет { a: 1, b: 2 }, переменная obj по
...
+// };
 
-console.log(notChangedObject);  // будет { a: 1, b: 2 } - поле 'a' сначала было 0, но потом мы записали поверх
+// console.log(notChangedObject);  // будет { a: 1, b: 2 } - поле 'a' сначала было 0, но потом мы записали поверх
 // поля объекта object, в которых 'a' было равно 1. Вот и в итоговом объекте 'a' равно 1.
 
 /*
  * Почитайте вот эту статью, начиная с абзаца 'Объект литералы'
- * https://medium.com/@stasonmars/%D0%BE%D0%BF%D0%B5%D1%80%D0%B0%D1%82%D0%BE%D1%80-spread-%D0%B8-rest-%D0%BF%D0%B0%D1%80%D0%B0%D0%BC%D0%B5%D1%82%D1%80%D1%8B-%D0%B2-javascript-22eb33cb0825#6846
+ * https://is.gd/jlpaNV
```
Comment (SergeiReutov) [inline Story-2/Subtask-2.js]: :-D
